src/deparam.js

```diff
@@ -52,7 +52,7 @@ function deparam(qs, coerce) {
     coerce = setDefault(coerce, true);
     qs = qs.substring(qs.charAt(0) === '?');
     const queryParamList = qs.split('&');
-    const queryObject = {};
+    const queryObject = Object.create(null);
```

Comment:
@scssyworks was there a reason to revert this?

Reply:
At least this one makes a difference:

```javascript
deparam('__proto__[a]=b'); // Will pollute prototype
```

Reply:
@sabberworm Apologies! Seems like it's a mistake on my part. I believe you have already raised an MR to fix that. Thank you for that!